libs/api/domains/vehicles/src/lib/services/vehicles.service.ts

```diff
@@ -530,10 +543,23 @@ export class VehiclesService {
     }
 
     try {
-      const dtos = await this.getMileageWithAuth(auth).rootPut({
+      const res = await this.getMileageWithAuth(auth).rootPutRaw({
         putMileageReadingModel: input,
       })
-      return dtos ?? null
+
+      if (res.raw.status === 204) {
+        this.logger.info('mileage update successful')
```

Comment:
Do you think this will be too noisy when all users have to register? Just a thoguht 🤷🏼

Reply:
relegated to debug